javascript/chronometer.js

```diff
@@ -49,9 +47,9 @@ class Chronometer {
   }
 
   split() {
-    let minutes= this.computeTwoDigitNumber(this.getMinutes())
-    let seconds= this.computeTwoDigitNumber(this.getSeconds())
-    return `${minutes}:${seconds}`;
+    let updatedMinutes= this.computeTwoDigitNumber(this.getMinutes(this.currentTime));
+    let updatedSeconds= this.computeTwoDigitNumber(this.getSeconds(this.currentTime));
```

Comment:
if you look at the getMinutes() method above, its not expecting any arguments, which means you should not be assing any when calling the function